infrastructure-playbooks/rolling_update.yml

```diff
@@ -110,10 +101,21 @@
         enabled: yes
       when: is_systemd
 
-    - name: select a running monitor
+    - name: set mon_host_count
+      set_fact: mon_host_count={{ groups.mons | length }}
+
```

Comment:
Oops I missed that, yes we cannot do a prompt on a `when`, a debug message seems enough then.